package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "echo 'Warning: TODO - Define Linting.'",
     "pretest-ci": "npm-run-all -s lint bootstrap",
     "seed": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedChallenges",
```

Comment:
```suggestion
    "seed": "npm-run-all -s seed:*",
    "seed:curriculum": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedChallenges",
```

Reply:
This is just to be able to run all the commands in tandem with [`npm-run-all`](https://www.npmjs.com/package/npm-run-all)